### ISSUE TITLE
Download subtitles automatically?

### ISSUE THREAD

**Issue by deadcoder0904** [mayeaux/generate-subtitles#61]
After I upload my file, I don't want a video to be played. Nor any other options.

I just want my subtitles downloaded.

Can you make it so that just the subtitles are downloaded?

Its a waste of bandwidth to stream the video again & for some reason, subtitles aren't easily downloadable either after converting.

I have to wait a long time even though the conversion is already done.

Why can't it just downloads automatically after conversion is done? It'll be a better UX too.

Why so much friction?

**Comment by gbonnema** [comment]
Hi deadcoder0904, it is not a waste of bandwidth. You don't need to play the video. It is a larger webpage than just the video. If you scroll down, you will find several download links, like *.srt, *.vtt and so on. So, the page contains everything you need to download, or play the video with subtitles, whichever you want.  

So, just scroll down, and you will find what you required. I am a user too, so I understand the confusion.

P.S: I mailed in reply, but apparantly that did not arrive in the github issue, so, just to be sure, I submit the reply through github.

Good luck in your attempts. Let me know if you have any issues.

**Comment by deadcoder0904** [comment]
no, i did see that i have to scroll down. that's how i download the `.srt` files.

i'd just like that to be fast.

and yes it does autoplay videos which is annoying.

also, fills up the whole screen with video which is not needed at all.

i think the only goal of freesubtitles is 2 pages. first page is upload video. 2nd page is download `.srt` or any other format like `.vtt`

but i do think most people are just using it as upload video & download `.srt` file. you can track it if you want with something like mixpanel or another free service like posthog or something. im pretty sure those use-cases are popular.

my only gripe is it covers the whole page with the video. meanwhile, i dont even need a video or any other things. just .srt file.

if there's a way to add preferences locally using local-storage for each user then it'd be wonderful. i'll probably use it to hide the video & bunch of other things.

all i do is download `.srt` file & delete transcription.

**Comment by gbonnema** [comment]
akshay kadam (a2k) kirjoitti 10.10.2023 klo 13.08:
>  > no, i did see that i have to scroll down. that's how i download the > 
|.srt| files. > > i'd just like that to be fast. > Ok, fair enough. 
Personally, I don't really care about the video ye or ne. I sometimes 
watch it to check how successful the transcription was. But I usually 
download srt and txt because those are the files I use.

Thanks for the explain!

Kind regards, Guus
